test/extended/images/helper.go

```diff
@@ -16,3 +24,52 @@ func GetImageLabels(c client.ImageStreamImageInterface, imageRepoName, imageRef
 	}
 	return image.Image.DockerImageMetadata.Config.Labels, nil
 }
+
+// ModifySourceCode will modify source code in the pod of the application
+// according to the sed script.
+func ModifySourceCode(oc *exutil.CLI, selector labels.Selector, sedScript, file string) error {
+	pods, err := exutil.WaitForPods(oc.KubeREST().Pods(oc.Namespace()), selector, exutil.CheckPodIsRunningFunc, 1, 120*time.Second)
+	if err != nil {
+		return err
+	}
+	if len(pods) != 1 {
+		return fmt.Errorf("Got %d pods for selector %v, expected 1", len(pods), selector)
+	}
+
+	pod, err := oc.KubeREST().Pods(oc.Namespace()).Get(pods[0])
+	if err != nil {
+		return err
+	}
+	return oc.Run("exec").Args(pod.Name, "-c", pod.Spec.Containers[0].Name, "--", "sed", "-ie", sedScript, file).Execute()
+}
+
+// GetPageCount makes a http request for an example application and returns the page count
+func GetPageCount(oc *exutil.CLI, endpoint string, re *regexp.Regexp) (int, error) {
```

Comment:
Would it be better to swap the regexp with a string ? Dont se any benefit of having a regular here, since we would like to reuse this method also for other framework